Content.Client/UserInterface/ServerInfo.cs

```diff
@@ -23,14 +23,22 @@ public ServerInfo()
 
             var uriOpener = IoCManager.Resolve<IUriOpener>();
 
-            var discordButton = new Button {Text = Loc.GetString("Join us on Discord!")};
+            var discordButton = new Button {Text = Loc.GetString("Discord")};
             discordButton.OnPressed += args => uriOpener.OpenUri(UILinks.Discord);
 
             var websiteButton = new Button {Text = Loc.GetString("Website")};
             websiteButton.OnPressed += args => uriOpener.OpenUri(UILinks.Website);
 
+            var reportButton = new Button { Text = Loc.GetString("Report Bugs") };
+            reportButton.OnPressed += args => uriOpener.OpenUri(UILinks.BugReport);
+
+            var creditsButton = new Button { Text = Loc.GetString("Credits") };
+            creditsButton.OnPressed += args => new CreditsWindow().Open();
```

Comment:
Should these be unsubscribed?

Reply:
What does this mean?

Reply:
The += subscribes to the event and -= would unsubscribe.

If you do a subscription without an unsubscription it can memory leak in certain circumstances. If both of them hang around for the life of the program it wouldn't matter so I'd have to check if the LobbyGui is disposed every time.